setup.cfg

```diff
@@ -31,17 +34,17 @@ project_urls =
 url = https://github.com/reactive-firewall/multicast.git
 
 [bdist_wheel]
-universal=1
+universal=0
 
 [options]
 zip_safe = false
 py_modules = multicast
 test_suite = tests
-python_requires = >=3.7, !=3.13.*
+python_requires = >3.9, !=3.14.*
 setup_requires =
-  setuptools>=45.0.0
-  wheel>=0.37.0
-  build>=1.1.1
+  setuptools>=75.0.0
+  wheel>=0.44.0
+  build>=1.2.1
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.